searchlib/src/vespa/searchlib/attribute/imported_search_context.cpp

```diff
@@ -95,6 +92,12 @@ TargetResult
 getTargetResult(ReverseMappingRefs reverseMappingRefs,
                 const ReverseMapping &reverseMapping,
                 SearchContext &target_search_context,
+                uint32_t committedDocIdLimit) __attribute__((noinline));
+
+TargetResult
+getTargetResult(ReverseMappingRefs reverseMappingRefs,
```

Comment:
We should have a separate getTargetFilterResult without weights and without unpack() when not propagating weights to synthetic posting list.

Reply:
Agree.